main.go

```diff
@@ -195,6 +213,26 @@ func runAPI(cfg *core.Config, dbi *gorp.DbMap, team core.AssetManagerTeam, provi
 		AllowedHeaders: []string{"Content-Type", "User-Agent", "X-Auth-Token"},
 	}).Handler(handler)
 
+	//Start audit logging.
+	rabbitQueueName := os.Getenv("CASTELLUM_RABBITMQ_QUEUE_NAME")
+	if rabbitQueueName != "" {
+		var rabbitUserInfo string
+		if rabbitUsername := os.Getenv("CASTELLUM_RABBITMQ_USERNAME"); rabbitUsername != "" {
+			rabbitUserInfo += rabbitUsername
+		}
+		if rabbitPass := os.Getenv("CASTELLUM_RABBITMQ_PASSWORD"); rabbitPass != "" {
+			rabbitUserInfo += ":" + rabbitPass
+		}
+		if rabbitUserInfo != "" {
+			rabbitUserInfo += "@"
+		}
+		rabbitHost := envOrDefault("CASTELLUM_RABBITMQ_HOSTNAME", "localhost")
+		rabbitPort := envOrDefault("CASTELLUM_RABBITMQ_PORT", "5672")
+
+		rabbitURI := fmt.Sprintf("amqp://%s%s:%s/", rabbitUserInfo, rabbitHost, rabbitPort)
+		api.StartAuditLogging(rabbitURI, rabbitQueueName)
```

Comment:
For the record: This would be the same issue as above, but I'm okay with it here since `amqp.Dial()` takes a string argument, so there's nothing we can do here.